samples/batch/transcription-enabled-storage/Connector/Serializable/Transcription/TranscriptionProperties.cs

```diff
@@ -0,0 +1,17 @@
+﻿// <copyright file="TranscriptionProperties.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace Connector
+{
+    public class TranscriptionProperties
```

Comment:
looks weird, TranscriptionProperties contains only one member TranscriptionError.

Reply:
Same as above: is used for deserialization/serialization.